readme.md

```diff
@@ -171,8 +171,8 @@ Below is a table showing the mapping of the LTS branches to the packages release
 
 | Release | Github Branch | LTS Status | LTS Start Date | Maintenance End Date | Removed Date |
 | :-----------: | :-----------: | :--------: | :------------: | :------------------: | :----------: |
-| [2020-1-31](https://github.com/Azure/azure-iot-sdk-csharp/tree/lts_2020-1-31_patch1) | lts_2020_01   | Active     | 2020-01-31     | 2020-06-30           | 2020-12-31   |
```

Comment:
@sharmasejal Once we have confirmation on these timelines, could you share the pictorial representation that you have in your slides? I'd like to update the "Planned release schedule" image below.

Reply:
@elhorton For review.